audb/core/publish.py

```diff
@@ -232,8 +232,10 @@ def job(file):
         task_description='Find media',
     )
     if update_media:
+        update_media = sorted(update_media, key=lambda x: x[0])
```

Comment:
Would be nice to add a comment before `if update_media` to indicate why and by which key we are sorting, e.g.
```python
# Add updated and new media to dependencies
# and sort them by paths
# as they can be returned in random order
# by the discovery job
```